src/main/java/de/qabel/desktop/Kernel.java

```diff
@@ -192,7 +193,7 @@ private ClientDatabase getConfigDatabase() {
                 try (Statement statement = connection.createStatement()) {
                     statement.execute("PRAGMA FOREIGN_KEYS = ON");
                 }
-                ClientDatabase clientDatabase = new DesktopClientDatabase(connection);
+                ClientDatabase clientDatabase = new ChatClientDatabase(connection);
```

Comment:
This should have been in a different PR, but whatever. The ChatClientDatabase is the DesktopClientDatabase with 2 more migrations needed for the chatserver when there are incoming shares.